oat-sa/extension-tao-itemqti-pci
# Feature/tr 5440 dispatch trace events

Related to https://oat-sa.atlassian.net/browse/TR-5440

- Add `play`, `stop`, `record` and `reset` events.

Test:
- Create an item with that PCI, publish it and check logged events.
- Check migration script with: `php index.php 'oat\tao\scripts\tools\Migrations' -c migrate -v 'oat\qtiItemPci\migrations\Version202308031344181465_qtiItemPci'`
- Play with it in playground. Run `npx grunt connect:dev:keepalive qtiitempcitest` from `/tao/views/build`
